SuperBench/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Testing variables
+SUPERBENCH_DATASET_DIRLAB = ""
+SUPERBENCH_DATASET_SPREAD = ""
+
+for( SUPERBENCH_DATASET IN )
+
+BlueprintPointer blueprint = Blueprint::New();
+SuperElastixFilter::Pointer = SuperElastixFilter::New();
+SuperElastixFilter->SetInput( image );
+SuperElastixFilter->SetLogger( logger );
+SuperElastixFilter->Set( blueprint );
+SuperElastixFilter->Update();
+
+logger->AddOutLogger();
```

Comment:
AddOutLogger() does not exist. This file seems to be a draft?

Reply:
Yes seems like this was committed by accident.